paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettings.kt

```diff
@@ -47,7 +50,23 @@ internal class PlaygroundSettings private constructor(
     fun updateConfigurationData(
         updater: (PlaygroundConfigurationData) -> PlaygroundConfigurationData
     ) {
-        _configurationData.value = updater(_configurationData.value)
+        val configurationData = updater(_configurationData.value)
+
+        displayableDefinitions.value.forEach { definition ->
```

Comment:
What is this code doing? And why is it needed now?

Reply:
Discussed on Slack, resets definition value if the selected option not applicable to the selected integration type to the first available option.